src/vanilla/GeneratorSettingsJs.cs

```diff
@@ -219,17 +219,7 @@ private static void Log(Category category, string message, params string[] messa
             }
             else
             {
-                switch (category)
-                {
-                    case Category.Debug:
-                    case Category.Information:
-                        Console.WriteLine($"{category}: {logMessage}");
-                        break;
-
-                    default:
-                        Console.Error.WriteLine($"{category}: {logMessage}");
-                        break;
-                }
+                Console.WriteLine($"{category}: {logMessage}");

```

Comment:
Note that stdout is used for jsonrpc communication with the host. So writing non-JSON messages might mess it up--if you want to see this output, I suggest writing to `Console.Error`.

Reply:
BTW, what does `IsActive` even mean in this context?

Reply:
IsActive means whether we are in a current autorest generation session or not. This function is really about distinguishing between CI unit tests (use Console) and actual autorest generation (use Logger.Instance).

Reply:
OK, now I think I get it..